thirdeye/thirdeye-frontend/app/pods/components/rootcause-data-indicator/component.js

```diff
@@ -39,7 +40,13 @@ export default Component.extend({
 
     fetch(`/rootcause/raw?framework=identity&urns=${selectedUrn}`)
       .then(checkStatus)
-      .then(res => setProperties(this, { maxTime: parseInt(res[0].attributes.maxTime[0], 10) }));
+      .then(res => {
+        if (_.isEmpty(res[0].attributes.maxTime)) {
```

Comment:
Are we sure `res` will always be an array of objects, and that it will always have an `attributes` property?

Reply:
we could just use a try catch if response from the back end are inconsistent

Reply:
yes. it's generated by the RCA formatters by default